src/diffusers_interpret/token_attributions.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def plot(self, plot_type: str = 'barh', title: str = 'Token Attributions', **plot_kwargs) -> None:
+    '''
+    Plot the normalized token attributes to have a comparative view.
+    Available plot types include bar chart, horizontal bar chart, and pie chart.
+    '''
+    tokens, attributions = list(
+        zip(*self))  # TODO: this can be changed, depending on how we construct the class
+
+    plot_kwargs = {'title': 'Token Attributions', **plot_kwargs}
+
+    plt.title(title)
+
+    if plot_type == 'bar':
+        # Bar chart
+        plt.bar(tokens, attributions)
+        plt.xlabel('tokens')
+        plt.ylabel('attribution value')
+
+    elif plot_type == 'barh':
+        # Horizontal bar chart
+        plt.barh(tokens, attributions)
+        plt.xlabel('attribution value')
+        plt.ylabel('tokens')
+        plt.gca().invert_yaxis()
+
+    elif plot_type == 'pie':
+        # Pie chart
+        plt.pie(attributions,
+                startangle=90,
+                counterclock=False,
+                #  explode = (attributions <= 3) * 0.5,
+                labels=tokens,
+                autopct='%1.1f%%',
+                pctdistance=0.8)
+
+    else:
+        raise NotImplementedError(
+            f"`plot_type = {plot_type} is not implemented. Choose one of: ['bar', 'barh', 'pie']")
```

Comment:
Great job! I changed your code to put this inside a `list` class and some other small improvements.
The idea is for this class to replace [this attribute](https://github.com/JoaoLages/diffusers-interpret/blob/3e65f811983cd62d4b5239c4d5f514c6e86101a3/src/diffusers_interpret/data.py#L60) of the output class. 
We can merge this PR and I'll do that change :)

```suggestion
from typing import Tuple, List, Any, Iterable, Union
import matplotlib.pyplot as plt


class TokenAttributions(list):
    def __init__(self, token_attributions: List[Tuple[str, float]]) -> None:
        super().__init__(token_attributions)
        self.token_attributions = token_attributions

    def __getitem__(self, item: Union[str, int]) -> Any:
        return getattr(self, item) if isinstance(item, str) else self.token_attributions[item]

    def __setitem__(self, key: Union[str, int], value: Any) -> None:
        setattr(self, key, value)

    def plot(self, plot_type: str = 'barh', title: str = 'Token Attributions', **plot_kwargs) -> None:
        '''
        Plot the token attributions to have a comparative view.
        Available plot types include bar chart, horizontal bar chart, and pie chart.
        '''
        tokens, attributions = list(zip(*self.token_attributions))

        # get arguments from plot_kwargs
        xlabel = plot_kwargs.get('xlabel')
        ylabel = plot_kwargs.get('ylabel')
        title = plot_kwargs.get('title') or title

        if plot_type == 'bar':
            # Bar chart
            plt.bar(tokens, attributions)
            plt.xlabel(xlabel or 'tokens')
            plt.ylabel(ylabel or 'attribution value')

        elif plot_type == 'barh':
            # Horizontal bar chart
            plt.barh(tokens, attributions)
            plt.xlabel(xlabel or 'attribution value')
            plt.ylabel(ylabel or 'tokens')
            plt.gca().invert_yaxis() # to have the order of tokens from top to bottom

        elif plot_type == 'pie':
            # Pie chart
            plot_kwargs = {
                'startangle': 90, 'counterclock': False, 'labels': tokens, 
                'autopct': '%1.1f%%', 'pctdistance': 0.8,
                **plot_kwargs    
            }
            plt.pie(attributions, **plot_kwargs)
            if xlabel:
              plt.xlabel(xlabel)
            if ylabel:
              plt.ylabel(ylabel)

        else:
            raise NotImplementedError(
                f"`plot_type={plot_type}` is not implemented. Choose one of: ['bar', 'barh', 'pie']"
            )
        
        # set title
        plt.title(title)
```